brightness.lua

```diff
@@ -67,7 +67,7 @@ backends.brightnessctl = {
         return tonumber(readcommand("brightnessctl --class=backlight max")) ~= nil
     end,
 
-    parse_output = function(self, output, callback)
```

Comment:
Seems this got lost in the refactoring.

Reply:
If you mean the callback param, that's because it was unused. Parse_output is an internal function, which is synchronous. So doesn't need a callback.